micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSView.js

```
@@ -73,6 +54,34 @@ const MDMSView = ({...props}) => {
+    },
+    config: {
+      enabled: data?.data?.sorId ? true : false,
+      select: (response) => {
+        return response?.mdms?.[0];
+      },
+    },
+    changeQueryName: uniqueIdentifier?.split(".")[0],
+  };
+  const { isLoading: isSORLoading, data: sorData } = Digit.Hooks.useCustomAPIHook(reqCriteriaSorSearch);
+
+  let propsToSendButtons = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'propsToSendButtons' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)